nf_core/modules/modules_repo.py

```diff
@@ -192,6 +192,7 @@ def setup_local_repo(self, remote, branch, hide_progress=True):
                         self.repo = git.Repo.clone_from(
                             remote,
                             self.local_repo_dir,
+                            filter=["tree:0"],
```

Comment:
Can we add a comment to warn what this is doing in case it breaks something in the future?